src/selectors/data/bookingsSelectors.js

```diff
@@ -124,13 +124,18 @@ export const selectCancelledBookings = createSelector(
   bookings => bookings.filter(booking => booking.isCancelled)
 )
 
-export const selectUsedBookings = createSelector(
-  state => state.data.bookings,
-  bookings => bookings.filter(booking => booking.isUsed)
+export const selectUsedThingBookings = createSelector(
+  selectBookings,
+  state => state.data.stocks,
```

Comment:
```suggestion
  selectStocks,
```
Si tu peux le faire sur les autres aussi ?

Reply:
J'avais voulu le faire, mais cela casse des tests. À mon avis pour une sombre question de dépendance circulaire que je n'arrive pas à tirer au clair...